cloudlift/config/service_configuration.py

```diff
@@ -257,6 +338,10 @@ def _validate_changes(self, configuration):
             "required": ["cloudlift_version", "services", "notifications_arn"]
         }
         try:
+            for _, service_configuration in configuration.get('services').items():
+                for sidecar in service_configuration.get('sidecars', []):
+                    if sidecar.get('name') == FLUENTBIT_FIRELENS_SIDECAR_CONTAINER_NAME and sidecar.get('log_driver') == 'awsfirelens':
+                        raise UnrecoverableException("Logging set to awsfirelens, but fluentbit firelens sidecar container is missing or has incorrect log driver.")
```

Comment:
This exception needs to be worded better

Reply:
reworded